hoodie-client/src/main/java/com/uber/hoodie/config/HoodieCompactionConfig.java

```diff
@@ -93,6 +95,10 @@
     // 200GB of target IO per compaction
     public static final String DEFAULT_COMPACTION_STRATEGY = LogFileSizeBasedCompactionStrategy.class.getName();
 
+    // used to merge records written to log file
+    public static final String DEFAULT_PAYLOAD_CLASS = HoodieAvroPayload.class.getName();
```

Comment:
can you double check DeltaStreamer & DataSource paths to see they use a different default? if so, that default needs to set on the `WriteClient` instantiation.

Reply:
thanks for pointing this out.